docs/src/TextField.doc.js

```diff
@@ -199,6 +205,53 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    id="tagsExample"
+    name="Example: Tags"
+    description={`You can include \`Tag\` elements in the input using the \`tags\` prop.`}
+    defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+  const [tags, setTags] = React.useState(['a@pinterest.com', 'b@pinterest.com']);
+  return (
+    <Box padding={2} color="white">
+      <TextField
+        id="emails"
+        label="Emails"
+        onChange={({value}) => {
+          const chunks = value.split(/[ ,;]+/);
+          if (chunks.length > 1) {
+            setTags([...tags, ...chunks.slice(0, -1)]);
+          }
+          setValue(chunks[chunks.length - 1]);
+        }}
+        onKeyDown={({event: {keyCode, target: {selectionEnd}}}) => {
+          if (keyCode === 8 /* Backspace */ && selectionEnd === 0) {
```

Comment:
I don't understand this, what does it do?

Reply:
This handles backspaces. If the cursor is in the beginning of the field and you press backspace, it removes the last tag. Should I add a comment?

Reply:
no, i should probably read this twice haha it was late tonight.